pkg/server/container_create_windows.go

```diff
@@ -541,6 +541,8 @@ func (c *criService) addOCIMounts(ctx context.Context, g *generator, platform im
 			}
 			src = formattedSource
 			mountType = "virtual-disk"
+		} else if strings.HasPrefix(src, `space://`) {
+			mountType = "ExtensibleVirtualDisk"
```

Comment:
Super nit: It looks like we chose to go with a "word1-word2-word3" format for the other types, maybe `extensible-virtual-disk` instead?

Reply:
When actually sending the request to HCS we need to use `ExtensibleVirtualDisk` so I had kept it as it is but that's not really a problem. I have changed it in order to follow the existing naming scheme.